modules/common-jpa-impl/src/main/java/org/opencastproject/job/jpa/JpaJob.java

```diff
@@ -198,7 +208,7 @@ public Date getDateStarted() {
   @Column(name = "dispatchable")
   private boolean dispatchable;
 
-  @Column(name = "job_load")
+  @Column(name = "job_load", nullable = false)
```

Comment:
Default value `1.0` missing

Reply:
Generating that reliably for multiple databases is unfortunately not possible but `nullable = false` plus setting a default value should be nerly identical. Fixed that.